egs2/kising/svs1/conf/tuning/train_visinger.yaml

```diff
@@ -0,0 +1,202 @@
+# This configuration is for ESPnet2 to train VITS, which
+# is truely end-to-end text-to-waveform model. To run
+# this config, you need to specify "--svs_task gan_svs"
+# option for svs.sh at least and use 22050 hz audio as
+# the training data (mainly tested on LJspeech).
+# This configuration tested on 4 GPUs (V100) with 32GB GPU
+# memory. It takes around 2 weeks to finish the training
+# but 100k iters model should generate reasonable results.
+
+##########################################################
+#                  SVS MODEL SETTING                     #
+##########################################################
+svs: vits
+svs_conf:
+    # generator related
+    generator_type: visinger
+    vocoder_generator_type: hifigan # hifigan, avocodo, uhifigan, visinger2
+    generator_params:
+        hidden_channels: 192
+        spks: 31
```

Comment:
I remember that the kising v2 has 33 singers, consider change it to 34 (with 1 additional placeholder)